src/components/layout/Panel.tsx

```diff
@@ -94,7 +95,7 @@ const Panel = function Panel({
     >
       <CardHeader onClose={onClose} fullWidth={scrollable || fullWidthHeader}>
         {Icon && <Icon className="w-em h-em" />}
-        <CardTitle>{title}</CardTitle>
+        <CardTitle classes={classnames({ grow: !!onClose })}>{title}</CardTitle>
```

Comment:
I have added the class only hen `onClose` is provided, but maybe `classes="grow"` would be enough.